app/controllers/api/cloud_volume_types_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  class CloudVolumeTypesController < BaseController
+    def cloud_volume_types_query_resource(object)
```

Comment:
What is this method for?  Are `CloudVolumeTypes` related to other `CloudVolumeTypes`?